.github/workflows/test_frameworks.yml

```diff
@@ -416,3 +416,31 @@ jobs:
           sed -i'' "s/test_langs/langs/g" tests/test_units/test_basic_app.py
       - name: Run tests
         run: nosetests
+
+  gunicorn-testsuite-20_1_0:
+    name: gunicon 20.1.0
+    runs-on: "ubuntu-latest"
+    env:
+      DD_TESTING_RAISE: true
+      # PYTHONPATH: ../ddtrace/tests/debugging/exploration/
```

Comment:
should we uncomment this line?

Reply:
Not at this time unfortunately 🙁 . Exploration tests cause the test suite run to freeze, which I think is what we're trying to fix in the long-term.